packages/docusaurus-plugin-content-docs/src/metadata.ts

```diff
@@ -145,6 +150,7 @@ export default async function processMetadata({
     lastUpdatedBy,
     lastUpdatedAt,
     sidebar_label,
+    isOld,
```

Comment:
I think we should rather try to compute, for each doc, what's the permalink to the latest version of the same doc.

maybe we need a property like `latestPermalink`

Reply:
some docs may get deleted in newer version of doc

Reply:
In such case latestPermalink would be undefined, and you wouldn't add the link?

Ah that's right, you want to link to the docs root, not necessarily the same doc in another version, so this is probably overkill.